scripts/credhub_shell.sh

```diff
@@ -2,13 +2,16 @@
 
 set -eu
 
+if [ -z "${SSH_PATH}" ]; then
+  SSH_PATH="/Users/${USER}/.ssh/id_rsa"
```

Comment:
Could make this a one-liner like we do elsewhere in the code base: `SSH_PATH=${SSH_PATH:-"/Users/${USER}/.ssh/id_rsa"}` or something to that effect.